nameko/testing/pytest.py

```diff
@@ -155,89 +158,6 @@ def amqp_uri(rabbit_config):
     return rabbit_config[AMQP_URI_CONFIG_KEY]
 
 
-@pytest.yield_fixture(autouse=True)
-def fast_teardown(request):
```

Comment:
Fixture provides no speedup now that we don't wait to cancel consumers on `kill`.